packages/headless/src/app/headless-engine.ts

```diff
@@ -238,11 +239,18 @@ export interface Engine<State = SearchAppState> {
    * The redux store.
    */
   store: Store;
+  /**
+   * A function for headless to call to retrieve a refreshed access token.
+   */
   renewAccessToken: () => Promise<string>;
   /**
    * The logger instance used by headless.
    * */
   logger: Logger;
+  /**
+   * Adds the specified reducers to the store.
+   */
+  addReducers(reducerMap: ReducersMapObject): void;
```

Comment:
The argument is called `reducerMap` here and `reducers` on line 294. For consistency maybe they should be the same.